index.js

```diff
@@ -759,7 +760,8 @@ class Hypercore extends EventEmitter {
       block = b4a.from(block)
 
       if (this.encryption.compat !== this.core.compat) this._updateEncryption()
-      this.encryption.decrypt(index, block)
+      if (this.core.unencrypted) this.encryption = null
```

Comment:
if we're storing in the manifest then it shouldn't be possible to dynamically set unencrypted right?

Reply:
ahh sorry, this is in the case that we do:
```
const core = new Hypercore(unencryptedManifest)
const sess = core.session({ encryptionKey }
```

in which case, should we throw an error?

Reply:
ya its never dynamically set, we check it here for replication reasons as the manifest might arrive post-ready